vivisect/extensions/__init__.py

```diff
@@ -29,6 +30,9 @@ def loadExtensions(vw, vwgui):
             vw.vprint('Invalid VIV_EXT_PATH dir: %s' % dirname)
             continue
 
+        if dirname not in sys.path:
```

Comment:
accidentally whacked this in the extensions_improvements PR, and somehow missed that it was *not* superfluous.